### PR TITLE
Added listbox::clear_headers() and menubar::clear()

### PR DESCRIPTION
**Pull request by besh81**
Added listbox::clear_headers() to remove all the columns from the listbox
Added menubar::clear() to remove all the items from menubar

### CODE REVIEW

File: source/gui/widgets/listbox.cpp [hunk 1]
```diff
@@ -5319,6 +5324,14 @@ namespace nana
 			return pos;
 		}
 
+		void listbox::clear_headers()
+		{
+			internal_scope_guard lock;
+			auto & ess = _m_ess();
+			ess.header.clear();
```
Comment (cnjinhao) [inline source/gui/widgets/listbox.cpp]: I think it is better that all items are deleted as well

Comment (besh81) [reply source/gui/widgets/listbox.cpp]: Yes I think it's better too :+1:

Comment (cnjinhao) [reply source/gui/widgets/listbox.cpp]: So, would you like to update the PR? Im looking forward to it.

**Comment by besh81** [comment]
I've added this line:
`ess.lister.erase();`
that should erase all the items and categories.

File: source/gui/widgets/listbox.cpp (hunk 1 above)
Comment (besh81) [reply source/gui/widgets/listbox.cpp]: Added ess.lister.erase(); to erase all the items.

**Comment by cnjinhao** [comment]
Thanks for your PR!

**Pull request merged**